packages/shared/lib/services/user.service.ts

```diff
@@ -9,17 +9,9 @@ const VERIFICATION_EMAIL_EXPIRATION = 3 * 24 * 60 * 60 * 1000;
 
 class UserService {
     async getUserById(id: number): Promise<User | null> {
-        const result = await db.knex.select('*').from<User>(`_nango_users`).where({ id });
+        const result = await db.knex.select('*').from<User>(`_nango_users`).where({ id }).first();
 
-        if (result == null || result.length == 0 || result[0] == null) {
-            return null;
-        }
-
-        if (result[0].suspended) {
-            return null;
-        }
-
-        return result[0];
+        return result && !result.suspended ? result : null;
```

Comment:
could the `suspended` be part of the query?

Reply:
true